HeavyFlavorAnalysis/RecoDecay/src/BPHPlusMinusCandidate.cc

```diff
@@ -73,6 +73,7 @@ void BPHPlusMinusCandidate::add(
                                           << "already containing same sign particle, add rejected";
         return;
       }
+      [[fallthrough]];
```

Comment:
@santocch @mrodozov would not a break be more correct? The cases look mutually exclusive to me

Reply:
- First call: no particles in the collection... so the routine just add the particle regardless of its charge...
   - Second call: check the charge (it has to be the opposite of the first added particle). If it's opposite then OK fallthrough --> add the particle. If it's the same charge then it's not OK and exit from the routine
   - Third call: too many particles --> EXIT

The routine's logic looks ok like it is now...